code/__HELPERS/mobs.dm

```diff
@@ -499,22 +499,31 @@ Proc for attack log creation, because really why not
 
 //Used in chemical_mob_spawn. Generates a random mob based on a given gold_core_spawnable value.
 /proc/create_random_mob(spawn_location, mob_class = HOSTILE_SPAWN)
-	var/static/list/mob_spawn_meancritters = list() // list of possible hostile mobs
-	var/static/list/mob_spawn_nicecritters = list() // and possible friendly mobs
-
-	if(mob_spawn_meancritters.len <= 0 || mob_spawn_nicecritters.len <= 0)
+	var/static/list/mob_spawn_meancritters // list of possible hostile mobs
+	var/static/list/mob_spawn_nicecritters // and possible friendly mobs
+	var/static/list/mob_spawn_neutralcritters // list of possible neutral mobs
+
+	if(!mob_spawn_meancritters)
+		mob_spawn_meancritters = list()
+		mob_spawn_nicecritters = list()
+		mob_spawn_neutralcritters = list()
 		for(var/T in typesof(/mob/living/simple_animal))
 			var/mob/living/simple_animal/SA = T
 			switch(initial(SA.gold_core_spawnable))
 				if(HOSTILE_SPAWN)
 					mob_spawn_meancritters += T
 				if(FRIENDLY_SPAWN)
 					mob_spawn_nicecritters += T
+				if(NEUTRAL_SPAWN)
+					mob_spawn_neutralcritters += T

```

Comment:
If you choose to have bits you'll want to replace this with say

`var/goldcorespawns = initial(SA.gold_core_spawnable)`
`>>>>if(goldcorespawns == NO_SPAWN)`
`>>>>>>>> continue //don't bother `
`>>>>if(goldcorespawns & HOSTILE_SPAWN)`
`>>>>>>>>mob_spawn_meancritters += T`
`>>>>if(goldcorespawns & FRIENDLY_SPAWN)`
`>>>>>>>>mob_spawn_meancritters += T`
`>>>>(same for neutral)`

Reply:
Alright, changed this as well.